es/02_0_Configurando_un_VPS_Bitcoin-Core.md

```diff
@@ -0,0 +1,28 @@
+
+# Capítulo dos: Creando un Servidor Privado Virtual de Bitcoin-Core
+
+Para iniciar con Bitcoin, se debe configurar una maquina ejecutando el software de Bitcoin. Los artículos en este capítulo describen como hacerlo, principalmente mediante el uso de un VPS (servidor privado virtual).
+
+
+## Objetivos de este capítulo
+
+Después de trabajar en este capítulo, un desarrollador podrá:
+
+   * Decidir entre los cinco tipos principales de nodos de Bitcoin
+   * Cree un nodo de Bitcoin para desarrollo.
+   * Cree una instancia local de la Blockchain de Bitcoin
+
+Los objetivos de apoyo incluyen la capacidad de:
+
+   * Comprender la configuración de red básica del VPS.
+   * Decidir qué prioridades de seguridad implementar.
+   * Comprender la diferencia entre nodos podados y no podados.
+   * Comprender la diferencia entre los nodos Mainnet, Testnet y Regtest.
+   * Interpretar los conceptos básicos del archivo de configuración de Bitcoin.
+   
+## Tabla de contenido
+
+En realidad, no es necesario leer este capítulo completo. Decida si desea ejecutar un StackScript para configurar un nodo en un VPS Linode (§2.2); o desea configurarlo en un entorno diferente, como en una máquina AWS o Mac (§2.3). Luego puede ir a la sección correspondiente. Información adicional sobre nuestras configuraciones sugeridas también se puede encontrar en el Apéndice I. 
+
+   * [Sección uno: Configuración de un VPS de Bitcoin Core con Bitcoin Standup](02_1_Configurando_un_VPS_Bitcoin-Core_StackScript.md)
+   * [Sección dos: Configuración de una máquina con núcleo de Bitcoin a través de otros medios](02_2_Configurando_Bitcoin_Core_Otros.md)
```

Comment:
- Change link: `[2.2: Configurando una máquina Bitcoin-Core por otros medios](02_2_Configurando_Bitcoin_Core_Otros.md)` as in #377